src/css/style.scss

```diff
@@ -607,4 +553,27 @@ nav {
 
 .dropdown-menus {
   padding-left: 35px;
+}
+
+
+@media only screen and (max-width: 900px) {
```

Comment:
Nice work on the media query! Do we have something for tablet or will our regular layout work?